Ginger/GingerCoreNET/RunLib/CLILib/CLIProcessor.cs

```diff
@@ -54,30 +55,30 @@ public void ExecuteArgs(string[] args)
                 args = newArgs;
             }
             
-            ParseArgs(args);                                        
+           await ParseArgs(args);                                        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)